backend/src/auth.js

```diff
@@ -6,6 +6,10 @@ const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
 // Local imports
 const UserCreds = require('./models/userCreds');
 
+// Global variables
+const PORT_URL_COMPONENT = process.env.PORT? `:${process.env.PORT}` : ':3000';
```

Comment:
Another way to do this line would be: `const PORT_URL_COMPONENT = process.env.PORT || ':3000';`

Reply:
For completeness, [here's an article](https://www.codereadability.com/javascript-default-parameters-with-or-operator/) that talks about the dangers of using the `||` operator to set default variables. 

I acknowledge it can be risky, but personally I find it a useful hack when used responsibly. :)

Reply:
Also PORT should probably be added to the `.env.example` file since you're relying on it here.

Reply:
Thanks for linking the article, good to see the pros/cons! Decided to use the `||` operator 👍